packages/ethereum-contracts/contracts/gov/SuperfluidGovernanceBase.sol

```diff
@@ -101,6 +101,20 @@ abstract contract SuperfluidGovernanceBase is ISuperfluidGovernance
         }
     }
 
+    function batchUpdateSuperTokenLogicCustom(
```

Comment:
We could also overload and generalize it to the logic.

```
address[] calldata customTokenLogics
```

if `customTokenLogics[i]` is zero, we apply the regular `updateSuperTokenLogic` otherwise.

Reply:
oh, so we need to update governance to all the network, right? Scary :)

Reply:
yeah. If we can't do it without governance update anyway, I'd go with the non-breaking change of leaving the previous methods and adding one.
In case of the host contract however it may be a good idea to replace with such a generalized version. It's a breaking change too, but since it can only be invoked by gov and they're updated (or not) together, it wouldn't break anything.